### DISCUSSION TITLE
how to use with serialport？

### DISCUSSION THREAD

**Discussion by Minori-ty** [cawa-93/vite-electron-builder#657] [Q&A]
I use vue-cli-electron with serialport, here is my demo:
https://github.com/Minori-ty/vue-cli-vue3-electron-serialport.git

but how to use whit serialport in this template?

**Comment by cawa-93** [comment]
Follow [the docs](https://github.com/cawa-93/vite-electron-builder#working-with-dependencies). If you found issue share minimal reproduction.
